### PR TITLE
GEOMESA-1395 Users should have access to a JAR bundle (.tar.gz) for G…

### PR DESCRIPTION
**Pull request by cne1x**
…eoMesa statistics for GeoServer

Creates a new .tar.gz bundle suitable for use with GeoServer,
provided the other dependencies have already been installed.

Signed-off-by: Chris Eichelberger <cne1x@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
might be good to standardize the gs-web installations somehow - we currently also have a geomesa-web/geomesa-web-install module. Longer term, it would probably be good to split out the spark-rest stuff into it's own module/install since it's a lot more complicated than the others.

File: geomesa-gs-plugin/geomesa-stats-gs-plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <!-- provided dependencies -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+            <version>3.1.0.RELEASE</version>
```
Comment (jnh5y) [inline geomesa-gs-plugin/geomesa-stats-gs-plugin/pom.xml]: Is this managed elsewhere?

Also, can you quickly run build/calculate-cqs.sh to see if anything changes CQ-wise?

Comment (jnh5y) [inline geomesa-gs-plugin/geomesa-stats-gs-plugin/pom.xml]: Actually, wait, are the provided dependencies listed to 'exclude' them from the tar.gz?

File: build/cqs.tsv [hunk 2]
```diff
@@ -37,6 +37,7 @@ com.typesafe.scala-logging:scala-logging_2.11	3.1.0	compile
 com.vividsolutions:jts	1.13	compile
 com.yammer.metrics:metrics-core	2.2.0	compile
 eu.medsea.mimeutil:mime-util	2.1.3	compile
+info.ganglia.gmetric4j:gmetric4j	1.0.7	compile
```
Comment (jnh5y) [inline build/cqs.tsv]: *blink* *blink*  What?!

**Pull request merged**